setup.py

```diff
@@ -124,16 +124,26 @@ def get_all_extras() -> Dict:
     long_description_content_type='text/markdown',
     packages=find_packages(include=["aea*"]),
     classifiers=[
+        'Environment :: Console',
+        'Environment :: Web Environment',
         'Development Status :: 2 - Pre-Alpha',
         'Intended Audience :: Developers',
-        'Natural Language :: English',
         'License :: OSI Approved :: Apache Software License',
+        'Natural Language :: English',
+        'Operating System :: MacOS',
+        'Operating System :: Microsoft',
+        'Operating System :: Unix',
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: 3.8',
+        'Topic :: Communications',
+        'Topic :: Internet',
+        'Topic :: Scientific/Engineering',
+        'Topic :: Software Development',
+        'Topic :: System',
     ],
     install_requires=base_deps,
-    tests_require=["tox"],
+    tests_require=['tox'],
```

Comment:
was this by purpose? If yes, why not changing the other strings below from "double-quoted" to 'single-quoted'?

Reply:
consider also the classifier strings above.